### ISSUE TITLE
Mention required free space

### ISSUE THREAD

**Issue by FireEmerald** [GameServerManagers/LinuxGSM#2338]
## User Story

As a improvement for the LinuxGSM [website](https://linuxgsm.com/servers/)(s), I want to mention that we should add the size of the files downloaded for a server to the **Installation Guide/Requirements** ([e.g. here](https://linuxgsm.com/lgsm/kf2server/)), so that it's mentioned anywhere, how much free space a server requires.

## Further Information

As an example the Killing Floor 2 server requires at least **18.32 GiB** of free space, as written on the [steamdb.info site](https://steamdb.info/app/232130/depots/).

It would be a first step as information for the user, without implementing a code-side check mentioned in #704.

**Comment by dgibbs64** [comment]
Ideally, if a pre-compiled list was made available I would simply need to add the data. If a volunteer is willing to gather the data into a spreadsheet that would speed this up drastically.

**Comment by dgibbs64** [comment]
Secondly, this can also be incorporated into the script itself

**Comment by Frisasky** [comment]
should we must taking space used by steamcmd in accounting?

**Comment by dgibbs64** [comment]
@Frisasky probably just easier to add 1GB to all calculations. That will probably be OK

**Comment by FireEmerald** [comment]
I don't think we should take steamcmd and temp files into account - they are so small compared with the server files itself:

![1](https://user-images.githubusercontent.com/1107939/57275272-4cf85e80-709e-11e9-838e-37aa4d0a4b05.png)

As the size of the server files **may change** from time to time, I think it would be the best to not hard-code the size. If we would do this, we must update the info from time to time.

How about something like:
![2019-05-07 08_28_31-Killing Floor 2 - LinuxGSM](https://user-images.githubusercontent.com/1107939/57276499-20dedc80-70a2-11e9-8408-d0db3c623dba.png)

**Comment by dgibbs64** [comment]
Yeah this is why I think maybe adding 1GB "breathing room" is a good idea. 

Ideally, this process should be automated using a GitHub repo and Travis to generate a csv file but that is a whole mini project.

**Comment by Frisasky** [comment]
so.... which method we decide to be used? manually or just posting a steamdb link?

**Comment by dgibbs64** [comment]
Ultimately I would like to add the info in to LinuxGSM itself to warn the user. So manually adding would be preferable with the steamdb link as well.

**Comment by dgibbs64** [comment]
Steps are being put in place to have minimum space requirements in code. This will eventually be put on to the website

**Comment by dgibbs64** [comment]
steamcmd.net api can be used to get max sizes for steamcmd servers
